src/main/java/bio/terra/service/job/JobService.java

```diff
@@ -64,6 +64,7 @@
     private final KubeService kubeService;
     private final AtomicBoolean isRunning;
     private final Migrate migrate;
+    private final GoogleResourceConfiguration resourceConfiguration;
```

Comment:
It doesn't look like `resourceConfiguration` is used.

Reply:
Thank you - nice catch! I've removed it.